tests/test_uncertain_conditional.py

```diff
@@ -64,9 +62,7 @@ def gen_L(n, *shape):
 
 
 def gen_q_sqrt(D_out, *shape):
-    return tf.convert_to_tensor(np.array(
-        [np.tril(rng.randn(*shape)) for _ in range(D_out)]),
-                                dtype=default_float())
+    return tf.convert_to_tensor(np.array([np.tril(rng.randn(*shape)) for _ in range(D_out)]), dtype=default_float())
```

Comment:
FWIW, I found the previous state more readable...

Reply:
To be honest the whole line is horrible and should be splitted into multiple variables :)